src/interface/cron/events/addedCollaborator.js

```diff
@@ -26,37 +27,33 @@ agenda.define(jobs.ADDED_COLLABORATOR, async (job, done) => {
       addedCollaborators.length
     );
     await processAddedCollaboratorEvents(addedCollaborators);
-    const lastAddedCollaboratorCheckpoint =
-      getLastAddedCollaboratorCheckpoint({ addedCollaborators, batchSize, latestBlockNumber });
-    if (lastAddedCollaboratorCheckpoint) {
-      await updateAddedCollaboratorCheckpoint(lastAddedCollaboratorCheckpoint);
-    }
     done();
   } catch (err) {
     console.error("Error in job", jobs.ADDED_COLLABORATOR, err);
     done(err);
   } finally {
+    if (addedCollaborators.length > 0) {

```

Comment:
This should be done before done is called in try block. Also lets not update the original logic